assets/src/edit-story/components/panels/preset/stylePreset/panel.js

```diff
@@ -62,6 +62,8 @@ const PresetButton = styled.button`
 const HighLight = styled.span`
   padding: 0 2px;
   ${({ background }) => generatePatternStyles(background)}
+  /* stylelint-disable-next-line */
+  -webkit-box-decoration-break: clone;
```

Comment:
Is this needed and not automatically added where relevant?

Reply:
It enlightened me now, it's only needed in the output.js (without stylelint-disable, the prefixed version will be replaced with the non-prefixed version and it will break the output style [something is not right with stylelint in that matter]).